ServerWhitelist.yml

```diff
@@ -2258,6 +2258,30 @@ AllowedPrefixes:
     - https://mega.nz/folder/YNUmWSSS#5W09Tobc6Y4k3i7XWZvMWQ/file/FIsV2RjQ #UBE 2.0 Lashivous Lashes
     - https://www.mediafire.com/file/6jq0ixmr9n45phc/UBE_2.0_Welkynar_Hussar_Armor.7z #UBE Conversion of Welkynar Hussar Armor Requires original from Nexus.
     - https://mega.nz/file/OPZw0TzB#Z66lmKqPjCJ3ZlWn4KbGHx8GnTOGcQkJynPynEjZfyo #UBE Patch for The Welkynar Knight Quest. 
+
+    #Craftian's Mods - Most will require manual download due to using a java script instead of direct file links
+    - https://craftian.itch.io/ #His Main Page
+    - https://craftian.itch.io/bad-moon #Badmoon
+    - https://craftian.itch.io/celestine #Celestine Main Page
+    - 1Fb0S8My7IXHprefjAbg5zDaVPPcw-si9 #Celestine Beyond Reach update Unvoiced
```

Comment:
This is probably a Google Drive ID, not an actual link?
You should cut paste this in the Google Drive ID section.